core/config/locales/en.yml

```diff
@@ -134,6 +134,10 @@ en:
         name: Name
         preference_source: Preference Source
         type: Provider
+      spree/price:
+        currency: Currency
+        amount: Price
+        is_default: Currently Valid
```

Comment:
"Currently valid" seems like a strange translation for `is_default` Boolean. I can't imagine what this does. "Currently valid" sounds like something date related. Do we plan to support time based pricing in the future?